dev/make/cmplr.clang.mk

```diff
@@ -29,7 +29,7 @@ CORE.SERV.COMPILER.clang = generic
 -DEBC.clang = -g
 
 COMPILER.mac.clang = clang++ -m64 -fgnu-runtime -stdlib=libc++ -mmacosx-version-min=10.14 -fwrapv \
-                     -Werror -Wreturn-type
+                     -Werror -Wreturn-type -falign-functions=16
```

Comment:
If this alignment issue is reproduced only locally, then PR might be merged without this change